src/openforms/scss/pdfs/_privacy-policy.scss

```diff
@@ -0,0 +1,13 @@
+.submission-privacy-policy {
+  display: flex;
+  align-items: flex-start;
+
+  &__text {
```

Comment:
for the future:

```scss
@use "microscope-sass/lib/bem";

.submission-privacy-policy {
  ...
  
  @include bem.element('text') { ... }
}
```

-> this spits out the correct strict selector `.submission-privacy-policy .submission-privacy-policy__text` instead of just `.submission-privacy-policy__text`